scripts/component_readme/readme_template.md

```diff
@@ -72,12 +86,24 @@ dataset.write(
         # "{{ argument.name }}": {{ (argument.type|eval)() }},
 {% endif %}
 {% endfor %}
-    }
+    },
+{% if is_consumes_generic %}
+    consumes={
+         <field_name>: <mapped_field_name>,
```

Comment:
```suggestion
         <field_name>: <dataset_field_name>,
```

Reply:
isn't it mapping the dataset field name to the component field name?

Reply:
nvm just checked the docs